composer.json

```diff
@@ -14,14 +14,14 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.6 || ^7.0",
```

Comment:
Sounds to me like this is not possible before 2.0... If @chirimoya also agrees we can tag this PR on the 2.0 milestone.